openproblems/tasks/_batch_integration/batch_integration_graph/methods/scanvi.py

```diff
@@ -5,10 +5,10 @@
 
 @method(
     method_name="scanvi",
-    paper_name="",
-    paper_url="",
-    paper_year=0,
-    code_url="",
+    paper_name="Deep generative modeling for single-cell transcriptomics",
+    paper_url="https://www.nature.com/articles/s41592-018-0229-2",
+    paper_year=2018,
```

Comment:
this is the scVI paper... scANVI came out in January in MSB:
name= Probabilistic harmonization and annotation of single‐cell transcriptomics data with deep generative models
url= https://www.embopress.org/doi/full/10.15252/msb.20209620
year= 2021